cmd/svcat/README.md

```diff
@@ -204,11 +204,13 @@ Bindings:
 ## Unbind all applications from an instance
 ```
 svcat unbind -n test-ns ups-instance
+deleted ups-binding
 ```
 
 ## Unbind a single application from an instance
 ```
 svcat unbind -n test-ns --name ups-binding
+deleted ups-binding
```

Comment:
Same as above

Reply:
yeah! totally forgot to change the rest of the block!